core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -393,8 +416,18 @@ public void init(Properties p) throws WorkloadException {
     String scanlengthdistrib =
         p.getProperty(SCAN_LENGTH_DISTRIBUTION_PROPERTY, SCAN_LENGTH_DISTRIBUTION_PROPERTY_DEFAULT);
 
-    int insertstart =
-        Integer.parseInt(p.getProperty(INSERT_START_PROPERTY, INSERT_START_PROPERTY_DEFAULT));
+  int insertstart =
```

Comment:
Is the spacing correct here? Do these statements align w/ those above (are there tabs in here?)